examples/find_package/FindPackage.cpp

```diff
@@ -0,0 +1,7 @@
+#include <SFML/Audio.hpp>
+#include <SFML/Graphics.hpp>
+#include <SFML/Network.hpp>
```

Comment:
Just including headers isn't really enough to verify that the linking actually worked. You add the include directory and include headers, but then not link anything and it will build fine, because you didn't use any of the symbols.

Suggest to instantiate some class from each module. It can be something very basic.

Reply:
That's fair. Can this be a nonsense program that just creates lots of objects and bares no resemblance to what real SFML applications may look like?

Reply:
At first I thought it was in the test directory, but if it's in the example directory, we should have something more or less useful, as they are examples after all 😅

Reply:
Alternatively, I could convert an existing example into an installation test. Would that be better?

Reply:
This does make me wonder why I didn't put these in the test/ directory. That makes more sense because I don't want this code used as an example. In that case I would be able to write nonsense code that maximizes testing potential without worrying about others trying to copy it.